### DISCUSSION TITLE
Budibase : self-hosting : starting app

### DISCUSSION THREAD

**Discussion by slfhstr** [Budibase/budibase#1355] [Help] [answered]
I am trying to selfhost Budibase on VPS.
Followed instructions
Added my hosting key in hosting.properties
Brought up instance with docker compose
What next ??
Visiting localhost:10000 just brings a screen with documentation links
Not the screen seen when using budibase hosting

Also budibase_couch starts but exits :
   {"error":"file_exists","reason":"The database could not be created, the file already exists."}
while app_service reports :
    error: 'not_found',
    app-service_1      |   reason: 'Database does not exist.'

**Comment by slfhstr** [comment, marked as answer]
Update for others with same problem of understanding :
Response on Discord clarifies that after installing self-host, the Builder app is the **native desktop downloaded thick-client**. 
I was searching for the Builder app in the self-host, trying different port numbers etc
With this penny drop realisation, all seems to be working.

Oh, and it would be remiss for me not to say : 
**Budibase is AWESOME !** Truly effing fantastic.  I can't believe how excellent system it is.  You guys rock !

**Reply by joebudi** [reply]
Hey!

Quick note. We are working on removing the electron/desktop app and providing a similar flow to your initial attempts.

This will come in the next few weeks.

Also, thanks for the kind words.

**Reply by slfhstr** [reply]
Ah ok, great !
Look forward to it

**Comment by ndlang** [comment]
I am having the same issue but I do not understand hat the fix for this is?
